src/main/resources/assets/immersivepetroleum/lang/es_es.json

```diff
@@ -118,12 +118,12 @@
 	
 	"_comment": "################## IE-Manual ##################",
 	
-	"manual.immersivepetroleum.main": "Procesado de petróleo",
+	"manual.immersivepetroleum.main": "§lProcesado de petróleo§r",
```

Comment:
Not a big fan of the category text being **fat**

Any specific reason for that?

Reply:
Hello @TwistedGate , yes, I am in responsible for the Spanish translation of immersive engineering, and the entries in the manual have been redone in bold, and to match the rest is why I put it here too, although it is not finished yet...

Reply:
@TwistedGate Should I fix it or can I keep it that way?

Reply:
> 
> 
> Hello @TwistedGate , yes, I am in responsible for the Spanish translation of immersive engineering, and the entries in the manual have been redone in bold, and to match the rest is why I put it here too, although it is not finished yet...

Alrighty

> 
> 
> @TwistedGate Should I fix it or can I keep it that way?

Keep it